cpp-client/README.md

```diff
@@ -1,10 +1,15 @@
-# Building the C++ client from a base Ubuntu 20.04 or 22.04 image
+# Building the C++ client on Ubuntu 20.04 / 22.04 and Windows 10 / 11.
 
-These instructions show how to install and run the Deephaven C++ client, its dependencies,
-and its unit tests. We have tested these instructions in Ubuntu 22.04 with the default
-C++ compiler and tool suite (cmake etc).  We have used the instructions in the past to build
+These instructions show how to install and run the Deephaven C++ client, its
+dependencies, its unit tests on Linux and Windows. We have tested these instructions in Ubuntu 22.04 with the default
+C++ compiler and tool suite (cmake etc). We have also tested these instructions
+on Windows 10 and 11 with Visual Studio Community Edition.
+We have used the instructions in the past to build
 for older Ubuntu versions (20.04) and for some Fedora versions, but we don't regularly test
-on them anymore so we do notguarantee they are current for those platforms.
+on them anymore so we do not guarantee they are current for those platforms.
+
+Instructions for Linux are below. Instructions for Windows are at the end of
+the document.
 
 1. Start with an Ubuntu 22.04 install
```

Comment:
Maybe it makes more sense to make this another section, like you did for windows?  Meaning "Build the C++ client for [linux/windows]" is each its own title/section so they are at the same level?

Reply:
Fixed